config/webpack.config.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const merge = require('webpack-merge');
+
+const common = require('./webpack.common.js');
+const PATHS = require('./paths');
+
+// Merge webpack configuration files
+const config = merge(common, {
+  entry: {
+    popup: PATHS.src + '/popup.js',
+    contentScript: PATHS.src + '/contentScript.js',
```

Comment:
I found this only works for me if both `contentScript` are changed to `contentscript`. Can you confirm this?

Reply:
I changed the spelling. I hope that's ok!

Reply:
That should be fine as it's now in proper camelcase. What's left is to rename the file `src/contentscript.js` to match. Thanks!